public/app/features/scenes/scenes/nestedBasicFromJson.tsx

```diff
@@ -0,0 +1,139 @@
+import { sceneFromJSON } from '../core/serialization';
+
+const model = {
+  key: '58ffac0b-62bf-4673-9409-eeeac921a19b',
+  title: 'Nested Scene demo (from JSON)',
+  data: 'TODO: serialize Scene data?',
+  inputs: [
+    {
+      key: '94d745c9-7c8e-4234-bd98-e620ee643a61',
+      inputParams: {},
+      type: 'SceneTimeRange',
+      range: {
+        from: 'now-6h',
+        to: 'now',
+      },
+    },
+    {
+      key: 'a3ebb3ff-dce8-40bd-8ef3-066fdbdeb7fa',
+      inputParams: {
+        timeRange: '94d745c9-7c8e-4234-bd98-e620ee643a61',
+      },
+      type: 'SceneDataProviderNode',
+      queries: [
+        {
+          refId: 'A',
+          datasource: {
+            uid: 'gdev-testdata',
+            type: 'testdata',
+          },
+          scenarioId: 'random_walk_table',
+        },
+      ],
+    },
+  ],
+  layout: {
+    root: [
+      {
+        direction: 'column',
+        key: '8d42de22-be9a-417e-b42c-634959ab7af4',
+        type: 'SceneFlexLayout',
+        children: [
+          {
+            $ref: 'b30b9a3e-7112-4279-be52-00a913d161a0',
+            type: 'NestedScene',
+          },
+        ],
+      },
+    ],
+    'b30b9a3e-7112-4279-be52-00a913d161a0': {
+      key: 'b30b9a3e-7112-4279-be52-00a913d161a0',
+      title: 'Inner scene',
+      data: 'TODO: serialize Scene data?',
+      inputs: [
+        {
+          key: 'a3ebb3ff-dce8-40bd-8ef3-066fdbdeb7fa',
+          inputParams: {
+            timeRange: '94d745c9-7c8e-4234-bd98-e620ee643a61',
+          },
+          type: 'SceneDataProviderNode',
+          queries: [
+            {
+              refId: 'A',
+              datasource: {
+                uid: 'gdev-testdata',
+                type: 'testdata',
+              },
+              scenarioId: 'random_walk_table',
+            },
+          ],
+        },
+        {
+          key: '94d745c9-7c8e-4234-bd98-e620ee643a61',
+          inputParams: {},
+          type: 'SceneTimeRange',
+          range: {
+            from: 'now-6h',
+            to: 'now',
+          },
```

Comment:
this is strange, the key is the same as an input (time range),  but it's also a layout child with the same key